core/block_test.go

```diff
@@ -21,61 +21,13 @@ func TestBlockHash(t *testing.T) {
 		chain  utils.Network
 		name   string
 	}{
-		{
-			// block 231579: goerli
-			// "https://alpha4.starknet.io/feeder_gateway/get_block?blockHash=0x40ffdbd9abbc4fc64652c50db94a29bce65c183316f304a95df624de708e746",
-			number: 231579,
-			chain:  utils.Goerli,
-			name:   "goerli network (post 0.7.0 with sequencer address)",
-		},
-		{
-			// block 156000: goerli
-			// "https://alpha4.starknet.io/feeder_gateway/get_block?blockNumber=156000",
-			number: 156000,
-			chain:  utils.Goerli,
-			name:   "goerli network (post 0.7.0 without sequencer address)",
-		},
-		{
-			// block 1: goerli
-			// "https://alpha4.starknet.io/feeder_gateway/get_block?blockNumber=1",
-			number: 1,
-			chain:  utils.Goerli,
-			name:   "goerli network (pre 0.7.0 without sequencer address)",
-		},
```

Comment:
are you sure that these tests should be removed? because sequencerFallback is part of our logic in old hashes, maybe there is an alternative in mainnet ?

Reply:
I've found already 2 mainnet examples in this test if I am not mistaken
https://github.com/NethermindEth/juno/blob/738356956709f4764b32ecbded7ce19e74311ac0/core/block_test.go#L31-L44